pkg/terraformer/config.go

```diff
@@ -261,21 +254,41 @@ func (t *terraformer) cleanupConfiguration(ctx context.Context) error {
 	return nil
 }
 
-// ensureCleanedUp deletes the job, pods, and waits until everything has been cleaned up.
+// ensureCleanedUp deletes the Terraformer pods, and waits until everything has been cleaned up.
 func (t *terraformer) ensureCleanedUp() error {
 	ctx := context.TODO()
```

Comment:
I think it makes sense that the `context` is passed by the caller, doesn't it?

Reply:
Yes, it makes sense for sure!